src/lib/statsig/statsig.tsx

```diff
@@ -89,7 +89,7 @@ export function toClout(n: number | null | undefined): number | undefined {
   }
 }
 
-const DOWNSAMPLE_RATE = 0.98 // 98% likely
+const DOWNSAMPLE_RATE = 0.99 // 98% likely
```

Comment:
```suggestion
const DOWNSAMPLE_RATE = 0.99 // 99% likely
```
Suggest updating comment to follow tweak